tedana/resources/config/outputs.json

```diff
@@ -27,25 +27,9 @@
         "orig": "betas_OC",
         "bidsv1.5.0": "desc-ICA_stat-z_components"
     },
-    "ICA accepted components img": {
-        "orig": "betas_hik_OC",
-        "bidsv1.5.0": "desc-ICAAccepted_components"
```

Comment:
Is this the first place we should add a new file name version code? That is we are changing file names and preventing people from saving files with these older names. We can rename the current ones something like "bidsv1.5.0old". While I don't think anyone would want to use these older names, the big benefit is, if someone has files with these old names, they'd/we'd be able to quickly remember what they now correspond to.

Reply:
BIDS is on v1.6.0 right now, so I'd lean toward adding that and changing the default over renaming v1.5.0. WDYT?

Reply:
I now realize that we should have done something like this for #736 as well... and we should probably have our different versions reflected in the outputs tables too!

Reply:
Aligning with v1.6.0 with this PR sounds sounds like a great solution.

Did #736 change any file names or just change what was in some of the files? If it changed a file name, I could see back-adding that into the output filename table, but this seems really minor. I'd say don't bother or consider it a really low priority.

Reply:
It changed the T2* and S0 maps. T2starmap.nii.gz --> desc-limited_T2starmap.nii.gz and desc-full_T2starmap.nii.gz --> T2starmap.nii.gz.

Reply:
Beyond adding the v1.6.0 naming convention (and reverting name changes, but not removals, from v1.5.0) in 280771c, here's what I'm thinking could help:

![image](https://user-images.githubusercontent.com/8228902/142033941-36f30a9c-c57c-49af-89ce-00062fc8a22e.png)

Unfortunately, this would involve working through our commit history to pin down when each filename was changed and which tedana versions the changes apply to, which I don't want to tackle in this PR, but I think the overall idea is solid.

Reply:
I suggest just opening a new issue for adding in the intermediate file naming convention.

FWIW, when we made this file naming decision, I think we talked about creating new file naming options that were linked to a released version. That is, we'd always be able to say "Save files as they were in tedana v0.0.9" I don't see that in the current code, but this discussion shows why such an option might be useful. Whatever we call each naming convention, there would also be an ordered list saying in which version each convention was the default. This wouldn't be too hard to add, but it would definitely be a new issue.

Reply:
@handwerkerd that is correct, I proposed that if this becomes an issue we could alias a few tedana versions to certain naming conventions (not track them in the JSON file itself).

Reply:
The logical place to do this would be in the input parser in the init of `io.OutputGenerator` The one quirk is that would  mean every time we edit outputs.json, we'd also want to add to the parser to peg the current version with the previous file names.